### DISCUSSION TITLE
The official ASA Stats pool(s) participation in the Yieldly's staking programs

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#102] [General]
The Keeper of the Development Pool yesterday supplied devpool's 1,229,711 TinymanPool1.1 ASASTATS-YLDY tokens to the [Yieldly staking program](https://algoexplorer.io/tx/group/2FFs5D9JOPV9u6DC2NexUN14S0TxWoeE3SWRbCEQW%2BI%3D). There's nothing about that to discuss officially as those funds are Team's and they have been [allocated to the LP](https://algoexplorer.io/tx/group/WKLaSQ2l1Y3UihYfNUnjnYONxPhjGcyasFWPzy6YiwU%3D) by the Team initiative.

First of all, the total cost in ASASTATS for the whole program is 13,166,558 + 6,583,279 and the Airdrop pool and the Administration Pool share the cost in half (Transparency reports [02/22](https://www.asastats.com/transparency-report-2022-02.pdf) and [03/22](https://www.asastats.com/transparency-report-2022-03.pdf)). Btw, the whole borrow/return process is not yet finalized, but it will eventually.

Based on the agreement, Yieldly's sent us 710,732 YLDY and together with our own 7,369,495 ASASTATS, we added those funds to the [Tinyman's ASASTATS-YLDY LP](https://algoexplorer.io/tx/group/EsnuWXJKcj2J7YJrr5bZhJInay9quQC2vOJiFOmLwRw%3D). It's worth noting that Yieldly's stance is that those funds should permanently stay in the liquidity pool.

First of all, the very first and the most important question is:

**should we stake those Tinyman's LP tokens in [Yieldly's staking program](https://app.yieldly.finance/liquidity-pools?id=751372353)?**

Just a note that I have asked for a sort of permission from Yieldl'y in the Telegram group and their answer is that nothing stops us from staking those tokens.

Another question is, who should be the owner of those 7,369,495 ASASTATS that are permanently locked in Tinyman's LP escrow? 

And the question that arises if we're about to stake those tokens, who would get the claimed YLDY rewards?

**Comment by Babbexx22** [comment]
Bearing in mind a constant (that ASA Stats has pools for whatever it wants to do; Admin pool, Airpool.. and the likes), I think we should reason out a way to channel the above mentioned funds (in gradual manner) into ASA Stats Algo liquidity pool. Since it appears to be the most traded.. We've done our best to improve the liquidity in that respect, but the situation still remains that it's not premium yet.
That's my take.

**Reply by ipaleka** [reply]
You're talking about channeling the funds. Does that imply a yes answer to the first question or you're talking about some other funds?

Btw, the admpool keeps borrowing from the devpool and that can't last forever - the admpool should start returning the funds...

**Reply by Babbexx22** [reply]
Yeah.. Yes.. We can go on with the staking.. Was just thinking of a way to strengthen the Algo ASA Stats LP the more. That's why I suggested the possibility of diverting the claimed reward to the aforementioned pool.

**Comment by AlgoRhythMatic** [comment]
I guess the only reason to not stake them:  allow for greater, undiluted returns for the community investors.  Otherwise, seems prudent to stake the tokens.

**Reply by ipaleka** [reply]
> I guess the only reason to not stake them: allow for greater, undiluted returns for the community investors. Otherwise, seems prudent to stake the tokens.

Exactly that, the first question actually is: would staking those tokens be an ethically correct move to take?

**Reply by Motuwagon** [reply]
I had similar question with staking my governance allocation.  Is it ethical to add the stats I obtained for free.  If the purpose is to bring in new people, or is the purpose to reward existing, or both.  Based on what was said about the yieldly Algo stake program it's intent is both rewarding existing while adding new people.  
The same argument can be made for this Asa stats adding to yieldly staking program Asa stats provides majority of the liquidity in the stats/ yieldly pool so the benefit should both encourage new providers but still rewarding existing.  Asa as a existing provider should benifit. How much is added in concern of fairness: perhaps adding enough maintaining  that payout percentage does not drop below a certain percent.  Or matching the highest community members supply.  
 But I'm for using the lp tokens and to distribute the returns to dev pool to pay back some of borrowed and the admin pool so it has a flexible budget hopefully won't need to borrow for future projects.

**Reply by ipaleka** [reply]
Thank you for the great comment! 

I should just add a remark that personally I would always advocate against any restrictions for any ASASTATS allocated from the official pools. As mentioned many times before, last time in the BlockRank discussion, all of the tokens allocated have proper reasoning for rewarding. And a matter of fact is that luck or anything similar doesn't affect the process.

That being said, your and other people's governance seat tokens are well deserved and on top of that, there's no institute of punishment in ASA Stats. Everybody should do what suits them the best.

**Reply by AlgoRhythMatic** [reply]
I agree with this perspective and was just about to comment on it elsewhere re: double-dip Algostake vs. Yieldly.  For this conversation, I would like to suggest a clear distinction between ASA Stats (the institution) and the investors, participants, and governors (the community).  I think it is fair game for a governor to use their ASASTATS as they see fit.  If a time came where a choice needed to be made between leaving staking in Yieldly or voting, then the governor will need to choose per their own ethics.  This is also the case for spending any governors tokens now vs. holding to vote.

In terms of whether or not to dilute the LP rewards - we could do a test and take a vote perhaps?  Measure APR before/after and see if the majority agreed with the dilution?

**Reply by ipaleka** [reply]
> If a time came where a choice needed to be made between leaving staking in Yieldly or voting

We are ASA Stats, tokens and values **associated** with the seat have to be eligible for voting. We are ready to create a [Chainlink network](https://github.com/asastats/dao-discussion/discussions/10#discussioncomment-2615557) for the purpose.

> In terms of whether or not to dilute the LP rewards - we could do a test and take a vote perhaps? Measure APR before/after and see if the majority agreed with the dilution?

My opinion is that we may use voting/polls for some less important things. For things like this, we simply have to comprehend the result of the discussion conducted. We need arguments and stances.

P.S. right now we have two members with confirmation and no member against.

**Comment by ipaleka** [comment]
This discussion hasn't outcome yet with clear advice, tho we haven't got any true opinion against participation. Also, this whole process brings a bit of urgency.

That being said, I'm about to stake around 54% of available LP tokens, precisely 1,229,711 LP tokens. That amount is chosen based on this discussion, but it's adjusted to equal the number of tokens staked by the devpool. So, the participation in rewards will be 50% (the existing 8,883.777 YLDY rewards available for claiming for the sake of simplicity won't be counted separately).

If this discussion outcomes in a negative answer in the future, then those LP tokens would be pulled out.

**Comment by bear1bear2bear3** [comment]
To me it seems a bit strange to add the LP tokens from ASAStats own pools to the staking pools. Because what would stop ASAStats from also adding some ASASTATS tokens to the Algostake pool? I do realise that theres a bit of a difference bc the LP brings value to everyone but its only a small step from staking LP to stating tokens in distribution pools.

On the other hand it is a valid argument to want to recoup some expenses.

Without knowing the exact cost and the estimated reward for staking i think we could stake so much LP that the expenses for algostake can be recouped for example? Or some % (that obviously needs to be decided on) of the overall costs so far.

**Reply by ipaleka** [reply]
> Because what would stop ASAStats from also adding some ASASTATS tokens to the Algostake pool?

That's a fair concern, but this differs as we did provide funds for Algostake rewards (as we did for the YLDY-ASASTATS and ASASTATS-ASASTATS staking programs' rewards), while we haven't provided rewards funds for this Yieldly's liquidity staking program - Yieldly's has provided those YLDY for rewards. And I asked Yieldly's officials about that aspect and they answered that they don't mind.

> Without knowing the exact cost and the estimated reward for staking i think we could stake so much LP that the expenses for algostake can be recouped for example?

The Administration Pool has got ASASTATS funds to cover all the expenses besides direct development costs (covered by the Development Pool). Right now those costs aren't significant, but soon after we start with a marketing campaign those will increase much. This opportunity is pretty rare, in the future the admpool will just sell its ASASTATS on open market. We have avoided doing that and used borrowing from the devpool as selling ASASTATS on Tinyman will affect the price.

Just to be clear, I as the Keeper of the Development Pool have got a single duty: secure funds for the ASA Stats development! Even if that implies dropping ASASTATS price ten times.

So, to avoid that we have to find sideways.

**Reply by wxivd** [reply]
> Because what would stop ASAStats from also adding some ASASTATS tokens to the Algostake pool?

I think the answer relies in the following critical information from @ipaleka 

> Based on the agreement, Yieldly's sent us 710,732 YLDY and together with our own 7,369,495 ASASTATS, we added those funds to the [Tinyman's ASASTATS-YLDY LP](https://algoexplorer.io/tx/group/EsnuWXJKcj2J7YJrr5bZhJInay9quQC2vOJiFOmLwRw%3D). It's worth noting that Yieldly's stance is that those funds should permanently stay in the liquidity pool.

If those funds should permanently stay in the Yieldly liquidity pool. Then it make sense for ASA Stats to stake not 54% only as is stated in this [comment](https://github.com/asastats/channel/discussions/102#discussioncomment-2815742) but the full 100%. 

Now the question regarding what shall ASA Stats utilise the rewards for? 

I agree with @Babbexx22 
> Yeah.. Yes.. We can go on with the staking.. Was just thinking of a way to strengthen the Algo ASA Stats LP the more. That's why I suggested the possibility of diverting the claimed reward to the aforementioned pool.

It is in the best interest of ASA Stats to utilise the rewards to further enhance the Algo LP pool.

**Reply by ipaleka** [reply]
An excerpt from the Transparency report for May:

- - - - -
On [Tue, 24 May 2022 23:17:28 UTC](https://algoexplorer.io/tx/group/zzlp8uWBZBQopa%2FcGtB7aCmvejzec5rP4YgzzUpZUo4%3D), the devpool added 1,229,711 TinymanPool1.1 ASASTATS-YLDY tokens on admpool's behalf. At that time, an amount of 8,883 YLDY rewards was available to claim.

On [Fri, 27 May 2022 19:26:32 UTC](https://algoexplorer.io/tx/group/W02KQsHFOvAN7rKSiogr72i4SzRIpemq66ymCEO4V88%3D) the Keeper claimed 50,560 YLDY rewards from the Yieldly's liquidity pool staking program.

An amount of 8,883 YLDY rewards are reserved for the devpool for the initial staking period. The remaining 41677 YLDY are shared equally among the devpool and the admpool.

On [Fri, 27 May 2022 19:41:13 UTC](https://algoexplorer.io/tx/6PUDAK46HTYBNIUTBWM27VP3MOI5HJHLDNM6WPBYJOK2BFEUSRJA) the devpool sent 20,838.5 YLDY to the admpool.

On [Fri, 27 May 2022 19:28:12 UTC](https://algoexplorer.io/tx/group/Q%2B615AJdy%2FOvgzD63h5MwRc0MD8wRm4a%2B4bRkdFla%2BQ%3D) the Keeper added the rest of 1,051,453 TinymanPool1.1 ASASTATS-YLDY tokens to the Yieldly's staking program.

The total amount staked in the Yieldly's liquidity pool is 3,510,876 ASASTATS/YLDY LP tokens. It is expected that the staking rewards in the future will be shared in the ratio of `1,229,711 : (1,229,711+1,051,453)` (around `35% : 65%`), respectively that the devpool will transfer 65% of all claimed rewards to the admpool in the future.

- - - - -
